src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating() {
```

Comment:
Header comment needed here. Try to enforce overriding by adding a keyword to the method signature. Exception is not necessary here.